Doc/library/calendar.rst

```diff
@@ -28,6 +28,92 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 2 BC, and so on.
 
 
+.. class:: Day
+
+   This enumeration defines the days of the week as integer constants.
+
+   .. attribute:: MONDAY
+
+      The constant for Monday, with a value of 0.
+
+   .. attribute:: TUESDAY
+
+      The constant for Tuesday, with a value of 1.
+
+   .. attribute:: WEDNESDAY
+
+      The constant for Wednesday, with a value of 2.
+
+   .. attribute:: THURSDAY
+
+      The constant for Thursday, with a value of 3.
+
+   .. attribute:: FRIDAY
+
+      The constant for Friday, with a value of 4.
+
+   .. attribute:: SATURDAY
+
+      The constant for Saturday, with a value of 5.
+
+   .. attribute:: SUNDAY
+
+      The constant for Sunday, with a value of 6.
+
+
+.. class:: Month
+
+   This enumeration defines the months of the year as integer constants.
```

Comment:
This could say:

`Enumeration defining months of the year as integer constants, from 1 to 12`

and avoid the need for verbose `the constant for January, with a value of 1`
(in other words, keep the `.. attribute:: JANUARY` lines but not their body)